services/us3/internal/cmd/meta.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	def "go.beyondstorage.io/v5/definitions"
+	"go.beyondstorage.io/v5/types"
+)
+
+var Metadata = def.Metadata{
+	Name:  "us3",
+	Infos: []def.Info{},
+	Pairs: []def.Pair{},
+	Factory: []def.Pair{
+		def.PairWorkDir,
+	},
+	Service: def.Service{
+		Features: types.ServiceFeatures{},
+	},
+	Storage: def.Storage{
+		Features: types.StorageFeatures{
+			WriteEmptyObject: true,
+
+			Create:   true,
+			Delete:   true,
+			Metadata: true,
+			List:     true,
+			Read:     true,
+			Stat:     true,
+			Write:    true,
```

Comment:
Maybe we should not change the default values of the features before implementing them?